Formula/clasp-cl.rb

```diff
@@ -0,0 +1,34 @@
+class ClaspCl < Formula
+  desc "Common Lisp implementation that brings Common Lisp and C++ together"
+  homepage "https://github.com/clasp-developers/clasp"
+  url "https://github.com/clasp-developers/clasp/archive/refs/tags/2.0.0.tar.gz"
+  sha256 "de3c5efaad0ca1e87ce58e937a45f2740d0c5cd869a500b606eb9e2ea37cb837"
+  license "GPL-2.0-or-later"
+  head "https://github.com/clasp-developers/clasp.git",
+       using:  :git,
+       branch: "main"
+
+  depends_on "boost" => :build
+  depends_on "libunwind-headers" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "sbcl" => :build
+  depends_on arch: :x86_64
+  depends_on "fmt"
+  depends_on "gmp"
+  depends_on "llvm@14"
```

Comment:
Does it not work with GCC or the macOS tools?

Reply:
I assume you are asking about LLVM? If so, no the dependency is on LLVM 14 specifically because we expose and use the LLVM IR interface in Lisp.

Reply:
And does it work with the latest LLVM?

Reply:
No. The latest LLVM has opaque pointers which we will have to adapt to. For some other Mac reasons (JIT) we may have to skip LLVM15 and upgrade when LLVM16 lands.

Because we rely on the LLVM API extensively it takes a lot to upgrade when they change that API. We track LLVM version as a major version number. In other words, LLVM15/16 will be Clasp 3.0.0 at a minimum.

Reply:
You should be able to disable opaque pointers (depending on how you use LLVM): https://llvm.org/docs/OpaquePointers.html#transition-state

Reply:
Thanks for the tip! I'll keep that in mind when start our LLVM15/16 upgrade.